paperqa/settings.py

```diff
@@ -239,6 +239,12 @@ class AgentSettings(BaseModel):
         default="gpt-4o-2024-08-06",
         description="Model to use for agent",
     )
+
+    agent_llm_config: dict | None = Field(
```

Comment:
A few lines below this there is an `agent_config` that is basically kwargs for the `Agent`/`ToolSelector`

Do you mind adjusting their names and/or descriptions so it's clear what is the difference between `agent_llm_config` and `agent_config`? They're similar enough in naming right now I think we should fix

Reply:
👍 Tried to make the descriptions more clear in distinction

Reply:
Thanks for your efforts! Being honest, it's still not quite intuitive enough for my tastes, but for now it's good.

I think we can make this clearer by moving `to_aviary_tool_selector` and `to_ldp_agent` to be methods of `AgentSettings`, since _all_ their information is derived from there.